build/babel.config.js

```diff
@@ -38,9 +38,9 @@ module.exports = function(mode) {
   const conf = mode === 'production' ? production : development;
 
   return {
-    'presets': conf.presets.concat([ '@babel/preset-react' ]),
-    'ignore': [
-      'node_modules',
+    presets: conf.presets.concat([ '@babel/preset-react' ]),
+    ignore: [
+      /node_modules\/(?!@qwant\/qwant-basic-gl-style)/,
```

Comment:
Do you think it would be a good idea to babelify all node module (except maybe mapbox-gl that provides a minified version already) ?

Reply:
Yeah, why not.
The price to pay is slower build times and a warning message from babel on `react-dom` because it is too big to be properly optimized. But we can exclude that one too.
Do you propose I do it here?

Reply:
Well, if multiple exceptions are also required the other way around, I am ok to merge this PR as is.  
And this problem should be less critical once IE11 becomes unsupported.

Reply:
Ok, I'm merging this then.